server/main.go

```diff
@@ -58,7 +58,9 @@ func main() {
 
 	r.HandleFunc("/project", mkHandler(ctx.getProjects)).Methods("GET")
 	r.HandleFunc("/project/{id}", mkHandler(ctx.getProject)).Methods("GET")
-	r.HandleFunc("/project/{id}/job", mkHandler(ctx.startBuild)).Methods("POST")
+	r.HandleFunc("/project/{id}/job", mkHandler(ctx.startStandardJob)).Methods("POST")
```

Comment:
Is there a way to extract the project from the hook payload ? this way the user has to set up only one endpoint for all the hooks instead of one hook per project ?